### PR TITLE
Make terraform cloud provider able to parse log including ansi codes

### LINKED ISSUE #674: Remove this -no-color flag after parsePlanResult supports parsing the message containing color codes.
https://github.com/pipe-cd/pipe/blob/e3a8f3156caf93290cb213b8e1de615ad99ad771/pkg/app/piped/cloudprovider/terraform/terraform.go#L109-L112

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #674

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/cloudprovider/terraform/terraform.go [hunk 1]
```diff
@@ -58,10 +88,13 @@ func (t *Terraform) Init(ctx context.Context, w io.Writer) error {
...
+	if t.options.noColor {
+		args = append(args, "-no-color")
+	}
+	for _, v := range t.options.vars {
 		args = append(args, fmt.Sprintf("-var=%s", v))
 	}
-	for _, f := range t.varFiles {
+	for _, f := range t.options.varFiles {
 		args = append(args, fmt.Sprintf("-var-file=%s", f))
 	}
```
Comment (khanhtc1202) [inline pkg/app/piped/cloudprovider/terraform/terraform.go:99]: nits, should we extract it as `t.makeCommandArgs` private method 🤔

**Comment by nakabonne** [comment]
With it we can see the full version of the preview result for Terraform 👍 
/lgtm

File: pkg/app/piped/cloudprovider/terraform/terraform.go (hunk 1 above)
Comment (nghialv) [reply pkg/app/piped/cloudprovider/terraform/terraform.go]: Sure. Applied that idea. 👍

**Comment by khanhtc1202** [comment]
Thx
/approve

**Pull request merged**